src/util.rs

```diff
@@ -41,6 +45,10 @@ pub(crate) trait MsKeyBuilder {
     where
         Pk: ToPublicKey,
         Ctx: ScriptContext;
+
+    /// All pushes in miniscript script construction will never exceed 2**32 bytes.
+    /// We only push slice of hashes outputs (32 bytes), and keys (33/65 bytes).
+    fn push_ms_slice(self, data: &[u8]) -> Self;
```

Comment:
I don't see this being called anywhere.

Reply:
Good catch. This is a remanent of previous design where I thought I could hide all `unwraps\expects` for miniscript library. But later on decided to have explicit unwraps at call site to avoid future errors.